ri_public_examples/download_files.py

```diff
@@ -10,21 +10,28 @@
 DEFAULT_CONFIG_PATH = "configs/stress_test_config.json"
 
 
-def _download_file(remote_path: str, local_path: Path) -> None:
+def _download_file(remote_path: str, local_path: Path) -> int:
     """Downloads file to local path."""
```

Comment:
nit: change to
"""Downloads file to local path and returns status code of request"""